packages/default/scss/menu/_layout.scss

```diff
@@ -91,41 +94,23 @@
     }
 
 
-    // Context menu
-    .k-popups-wrapper {
-        position: relative;
-        border: 0;
-        margin: 0;
-        padding: 0;
-    }
-    .k-menu.k-context-menu {
-        margin: 0;
-        padding: $popup-padding-y $popup-padding-x;
-        border-width: 1px;
-        border-style: solid;
-        box-sizing: content-box;
-    }
-    .k-animation-container .k-context-menu.k-menu-horizontal {
-        display: flex !important; // sass-lint:disable-line no-important
-    }
-
-
     // Orientation
     .k-menu-horizontal {
 
-        > .k-item + .k-item {
+        > .k-item + .k-item:not(.k-separator) {
```

Comment:
Instead of making the item selector more specific, use `.k-item.k-separator`. This will reduce the need to add more selectors to undo the changes that apply due to the high specificity.

Reply:
Separator selectors are improved